app/services/katello/puppet_class_importer_extensions.rb

```diff
@@ -3,32 +3,30 @@ module Services
     module PuppetClassImporterExtensions
       extend ActiveSupport::Concern
 
-      included do
-        def update_environment
-          change_types = %w(new obsolete updated)
-          changed = self.changes
+      def update_environment
+        change_types = %w(new obsolete updated)
+        changed = self.changes
 
-          change_types.each do |kind|
-            changed[kind].slice!(@environment) unless changed[kind].empty?
-          end
+        change_types.each do |kind|
+          changed[kind].slice!(@environment) unless changed[kind].empty?
+        end
 
-          #prevent the puppet environment from being deleted, by removing special '_destroy_' String
-          if changed['obsolete'][@environment]
-            changed['obsolete'][@environment] =
+        #prevent the puppet environment from being deleted, by removing special '_destroy_' String
+        if changed['obsolete'][@environment]
+          changed['obsolete'][@environment] =
               changed['obsolete'][@environment].select { |klass| klass != '_destroy_' }
```

Comment:
Rubocop failure: 

/var/lib/workspace/workspace/katello-pr-test/app/services/katello/puppet_class_importer_extensions.rb:17:15: C: Layout/IndentAssignment: Indent the first line of the right-hand-side of a multi-line assignment.

              changed['obsolete'][@environment].select { |klass| klass != '_destroy_' }

              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^